hybrid/compartment_examples/inter_comp_call/build/Makefile.inter_comp_call

```diff
@@ -31,6 +31,12 @@ clean:
 		rmdir bin --ignore-fail-on-non-empty; \
 	fi
 
+.SECONDEXPANSION:
```

Comment:
I'm not too familiar with advanced makefile usage. Is this `.SECONDEXPANSION` getting called implicitly somewhere? Additionally, perhaps it would be useful for me if you show me the sequence of commands to compile/execute one of these examples with this new infrastructure, and how the artifacts look like.

Reply:
Ok, let me rephrase. One question was "is `.SECONDEXPANSION` a `make` implicit" thing, which you said yes (complete with link). The other question I'll rephrase as "does this change anything in how we call `make`, or in what it produces"?

Reply:
>  The other question I'll rephrase as "does this change anything in how we call make, or in what it produces"?

`so you don't need to change your make invocation for it to happen` - in other words, it doesn't change anything in how we call make and it doesn't change any of the resulting artifacts.